platform-ajax/src/main/java/com/softicar/platform/ajax/document/service/AbstractAjaxDocumentService.java

```diff
@@ -45,11 +47,21 @@ protected void executePayloadCode(INullaryVoidFunction payloadCode) {
 		try {
 			payloadCode.apply();
 		} catch (Exception exception) {
-			document.getBody().executeAlert(IDisplayString.create(exception.getLocalizedMessage()));
+			IDisplayString message = getDisplayMessage(exception);
+			document.getBody().executeAlert(message);
```

Comment:
we could inline this 
```suggestion
			document.getBody().executeAlert(getDisplayMessage(exception));
```